persistence/persistence-jdbc/src/main/java/uk/gov/justice/services/jdbc/persistence/PreparedStatementWrapper.java

```diff
@@ -92,6 +92,22 @@ public int executeUpdate() throws SQLException {
         return result;
     }
 
+    public void setFetchSize(final int fetchSize) throws SQLException {
+        try {
+
+            final boolean autoCommitEnabled = preparedStatement.getConnection().getAutoCommit();
```

Comment:
This code is always in a transaction, so auto-commit is always false.

Reply:
As discussed, logging exception for this